WORKSPACE

```
@@ -118,6 +118,9 @@ nixpkgs_package(
     attribute_path = "curl",
     nix_file = "//nix:bazel.nix",
     nix_file_deps = common_nix_file_deps,
+    # Remove once we upgrade to Bazel >=3.0. Until then `nix-build` output
+    # confuses the JAR query in `sdk-daml-head`.
```

```suggestion
    # confuses the JAR query in `daml-sdk-head`.
```

Applies to all the changes below.

---

:facepalm: Sorry about that, fixed.